recipes/myst-nb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "myst-nb" %}
+{% set version = "0.9.2" %}
```

Comment:
0.10.0 is in PyPI with the license file

Reply:
CC @anirrudh

Reply:
thanks @isuruf @scopatz  - shall update!